spec/locales-currencies-tz.html

```diff
@@ -51,7 +50,7 @@ <h1>IsStructurallyValidLanguageTag ( _locale_ )</h1>
       </ul>
 
       <p>
-        The abstract operation returns true if _locale_ can be generated from the ABNF grammar in section 3.2 of the Unicode Technical Standard 35, or successor, starting with unicode_locale_id, and does not contain duplicate variant or singleton subtags (other than as a private use subtag). It returns false otherwise. Terminal value characters in the grammar are interpreted as the Unicode equivalents of the ASCII octet values given.
+        The abstract operation returns true if _locale_ can be generated from the EBNF grammar in section 3.2 of the Unicode Technical Standard 35, or successor, starting with unicode_locale_id, and does not contain duplicate variant or singleton subtags (other than as a private use subtag). It returns false otherwise. Terminal value characters in the grammar are interpreted as the Unicode equivalents of the ASCII octet values given.
```

Comment:
I guess ABNF or EBNF both work here, as Unicode defines both grammars for some reason.

Reply:
NO, we should surely switch to EBNF because Mark Davis is deleting ABNF from UTS35 now. The update will be pushed out very soon. See http://unicode.org/repos/cldr/trunk/specs/ldml/tr35.html#Unicode_Language_and_Locale_Identifiers
the ABNF was very buggy (see http://unicode.org/cldr/trac/ticket/11746 and more)
The EBNF on that latest version still have some minor problems and I am reviewing for Mark Davis now. He will fix them shortly.